src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -245,3 +252,121 @@ def _update_plot_stats(self, area: Area) -> None:
+        """
+        raise NotImplementedError
+
+
+class CoefficientDataExporter(BaseDataExporter):
+    def __init__(self, area: "CoefficientArea", scm_manager: "SCMManager" = None):
+        self._area = area
+        self._scm = scm_manager
+
+    @property
+    def labels(self) -> List:
+        return ["slot",
+                "avg trade rate [ct./kWh]",
+                "min trade rate [ct./kWh]",
+                "max trade rate [ct./kWh]",
+                "# trades",
+                "total energy traded [kWh]",
+                "total trade volume [EURO ct.]"]
+
+    @property
+    def rows(self) -> List:
+        from statistics import mean
+        if not self._scm:
+            return []
+        area_results = self._scm.get_area_results(self._area.uuid, serializable=False)
+        if not area_results["after_meter_data"]:
+            return []
+        trade_rates = [
+            t.trade_rate
+            for t in area_results["after_meter_data"]["trades"]
+        ]
+        trade_prices = [
+            t.trade_price
+            for t in area_results["after_meter_data"]["trades"]
+        ]
+        trades_energy = [
+            t.traded_energy
+            for t in area_results["after_meter_data"]["trades"]
+        ]
+        return [[self._area.past_market_time_slot,
+                mean(trade_rates) if trade_rates else 0.,
+                min(trade_rates) if trade_rates else 0.,
+                max(trade_rates) if trade_rates else 0.,
+                len(trade_rates) if trade_rates else 0.,
+                sum(trades_energy) if trades_energy else 0.,
+                sum(trade_prices) if trade_prices else 0.]]
+
+
+class CoefficientLeafDataExporter(BaseDataExporter):
+    def __init__(self, area: "CoefficientArea"):
+        self._area = area
+
+    @property
+    def labels(self) -> List:
+        if isinstance(self._area.strategy, SCMStorageStrategy):
+            return ["slot", "charge [kWh]", "offered [kWh]", "charge [%]"]
+        elif isinstance(self._area.strategy, (SCMLoadHoursStrategy, SCMLoadProfile)):
+            return ["slot", "desired energy [kWh]", "deficit [kWh]"]
+        elif isinstance(self._area.strategy, SCMPVStrategy):
+            return ["slot", "produced [kWh]", "not sold [kWh]"]
+        return []
+
+    @property
+    def rows(self):
+        slot = self._area.past_market_time_slot
+        if slot is None:
+            return []
+        if isinstance(self._area.strategy, SCMStorageStrategy):
+            s = self._area.strategy.state
+            return [[slot,
+                    s.charge_history_kWh[slot],
+                    s.offered_history[slot],
+                    s.charge_history[slot]]]
+        elif isinstance(self._area.strategy, (SCMLoadHoursStrategy, SCMLoadProfile)):
+            desired = self._area.strategy.state.get_desired_energy_Wh(slot) / 1000
+            # All energy is traded in SCM
+            return [[self._area.past_market_time_slot, desired, 0.]]
```

Comment:
Why adding the `deficit` column then? As you have a separate class, I suggest to remove it then. 
The same applies to the `not_sold` in the PV strategy, right?

Reply:
Admittedly I added both in order to have exactly the same generated files for both SCM and normal exchanges, however since you prefer to omit this I will do so, thanks!

Reply:
Ok, makes sense. 
Maybe others beg to differ, so maybe you can hold off this for now and wait for other opinions?